src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -1510,3 +1497,23 @@ def pick_strategies(strategy_pool,
 
   return StrategyInfo(fuzzing_strategies, arguments, additional_corpus_dirs,
                       extra_env, use_dataflow_tracing, is_mutations_run)
+
+
+def should_set_fork_flag(existing_arguments, strategy_pool):
+  """Returns True if fork flag should be set."""
+  # FIXME: Disable for now to avoid severe battery drainage. Stabilize and
+  # re-enable with a lower process count.
+  if environment.is_android():
+    return False
+  # Fork mode is not supported on Fuchsia platform.
+  if environment.platform() == 'FUCHSIA':
+    return False
+  # Fork mode is disabled on ephemeral bots due to a bug on the platform.
+  if environment.is_ephemeral():
+    return False
+  # Do not use fork mode for DFT-based fuzzing. This is needed in order to
```

Comment:
This comment about DFT is no longer relevant and should be removed right?

Reply:
Yeah I'll remove DFT stuff in another PR.